tests/p2p/conftest.py

```diff
@@ -30,3 +33,10 @@ def p2p_logger():
 
     return logger
 """
+
+
+@pytest.fixture
```

Comment:
Probably an `autouse=True` here?

Reply:
I find it hilarious that I forgot that.  Think I'll add a small test that verifies that asyncio is indeed patched.